components/RatChecker/RatChecker.tsx

```diff
@@ -112,9 +111,17 @@ function RatChecker() {
                 </Card.Header>
                 <Card.Body>
                     {!isChecking ? (
-                        <div>
+                        <div style={{display: "flex", alignItems: "center", flexDirection:"column"}}>
                             <p>Received a suspicious (or any other mod) that you want to check? Select it here and get quick results for known rats or legitimate mods (you can select multiple at once):</p>
-                            <Form.Control type="file" className={'form-control'} ref={ratFileInput} onChange={onFileUpload} multiple />
+                            <Form.Control type="file" className={'form-control'} style={{display: "none"}} ref={ratFileInput} onChange={(e)=>onFileUpload((e.target as HTMLInputElement).files)}multiple accept='.jar'/>
+                            <div className={styles.dNd} onClick={(e)=>ratFileInput.current?.click()} onDrop={(e)=>{
+                                onFileUpload(e.dataTransfer.files);
+                                e.preventDefault();
+                            }} onDragOver={(e)=>e.preventDefault()}>
+                                <div className={styles['dNd-content']}>
+                                    Drag and Drop Files or click to open file
+                                </div>
+                            </div>
                         </div>
```

Comment:
This look nice, but I would like to have the dNd-field be not as large (height). This seems unneccessary and depending on the screen size, the user cant instantly see the result without scrolling down.
I would like to have something like this (only hardcoded example):
![Uploading grafik.png…]()

Reply:
this may be a stupid q, but how do I view the png file, it just redirects me to https://github.com/Coflnet/hypixel-react/pull/1047/files

Reply:
Oh, no that was my mistake. Seems like I sent the comment, before the image was uploaded.
Image: 
![grafik](https://user-images.githubusercontent.com/58751503/217544580-93b9f498-a9d8-4ade-a1ca-43a13efddfde.png)

Reply:
Ok so rectangular
![image](https://user-images.githubusercontent.com/64080101/217550754-691a0766-9c85-48a0-b27b-da88ae382093.png)
This fine?

Reply:
yes